### PR TITLE
fix(helper): update vue3 helper

### PR DESCRIPTION
**Pull request by BibiSebi**
## What?

This PR updates the vue3 helper and the vue3 template. The scope of this PR includes:

- updating of the template to correctly use parseContent, and work with the new vue 3 helper
- removing field plugin provider from vue 3 helper
- adding reactive references to the useFieldPlugin hook (vue 3)

## Why?

JIRA: EXT-1919

### CODE REVIEW

**Comment by BibiSebi** [comment]
Current dependencies on/for this PR:
* main
  * **PR #260** <a href="https://app.graphite.dev/github/pr/storyblok/field-plugin/260" target="_blank">Graphite</a> 
    * **PR #266** <a href="https://app.graphite.dev/github/pr/storyblok/field-plugin/266" target="_blank">Graphite</a> 
      * **PR #267** <a href="https://app.graphite.dev/github/pr/storyblok/field-plugin/267" target="_blank">Graphite</a> 
        * **PR #269** <a href="https://app.graphite.dev/github/pr/storyblok/field-plugin/269" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/storyblok/field-plugin/269?utm_source=stack-comment).

**Comment by eunjae-lee** [comment]
Since @BibiSebi  you were the creator of this PR, I cannot assign you to review this, so just pinging you here :)

File: packages/cli/templates/vue3/src/components/FieldPluginExample/index.vue [hunk 1]
```diff
@@ -5,41 +5,34 @@ import Counter from './Counter.vue'
...
-    >
-      <svg
-        width="14"
-        height="14"
-        viewBox="0 0 12 12"
-        fill="none"
-        xmlns="http://www.w3.org/2000/svg"
-      >
+  <div v-if="plugin.type === 'loaded'">
+    <pre>{{ plugin }}</pre>
```
Comment (demetriusfeijoo) [inline packages/cli/templates/vue3/src/components/FieldPluginExample/index.vue:20]: Was it missed here @BibiSebi?

**Comment by eunjae-lee** [comment]
Sorry @demetriusfeijoo actually this PR was not fully ready. I marked it as ready for review, but Bibi still needs to work on it a bit more. Sorry for the confusion 🙌🏼

File: packages/cli/templates/vue3/src/components/FieldPlugin.vue [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <script setup lang="ts">
 import { useFieldPlugin } from '@storyblok/field-plugin/vue3'
 
-const plugin = useFieldPlugin()
+const plugin = useFieldPlugin({ parseContent: (content: unknown) => content })
```
Comment (BibiSebi) [inline packages/cli/templates/vue3/src/components/FieldPlugin.vue:4]: what should happen with this component?

**Review by eunjae-lee — APPROVED** [review]

**Comment by eunjae-lee** [comment]
### Merge Activity

* **Sep 14, 10:27 AM**: @eunjae-lee started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/storyblok/field-plugin/269).

**Pull request merged**